vnext/src/Libraries/Components/SafeAreaView/SafeAreaView.windows.js

```diff
@@ -60,4 +60,4 @@ if (Platform.OS === 'android' || Platform.OS === 'windows') {
   );
 }
 
-module.exports = exported;
+export default exported;
```

Comment:
same comment about changing export style as above, do we have to do this?

Reply:
Yes, the module is now a ES module, which changes how things are exported from it.